pkg/oras/client.go

```diff
@@ -235,6 +237,57 @@ func (c *Client) LoadModel(ref *oci.Reference) (*model.Model, error) {
 	return r.Model, nil
 }
 
+func (c *Client) Models() error {
+	refs, err := c.cache.ListReferences()
+	if err != nil {
+		return err
+	}
+
+	maxLen := []int{0, 0, 12, 20, 0}
+	for _, ref := range refs {
+		if len(ref.Repo) > maxLen[0] {
+			maxLen[0] = len(ref.Repo)
+		}
+		if len(ref.Tag) > maxLen[1] {
+			maxLen[1] = len(ref.Tag)
+		}
+
+		size := bts.ByteCountBinary(ref.Size)
+		if len(size) > maxLen[4] {
+			maxLen[4] = len(size)
+		}
+	}
+
+	format := "%-" + strconv.Itoa(maxLen[0]) + "s "  // repo name
+	format += "%-" + strconv.Itoa(maxLen[1]) + "s "  // tag
+	format += "%-" + strconv.Itoa(maxLen[2]) + "s "  // digest
+	format += "%-" + strconv.Itoa(maxLen[3]) + "s "  // create at
+	format += "%-" + strconv.Itoa(maxLen[4]) + "s\n" // size
+	fmt.Fprintf(c.out, format, "REPOSITORY", "TAG", "MODEL ID", "CREATED", "SIZE")
+
+	for _, ref := range refs {
+		timeDiff := time.Now().Sub(ref.CreatedAt)
+		timeShow := ""
+		if timeDiff.Hours()/(365*24.0*7) > 1.0 {
```

Comment:
Maybe we can find some third package to do the time format

Reply:
I think so, but I am not found, have any suggestion?

Reply:
Fixed